src/jdk.jartool/share/classes/sun/tools/jar/resources/jar_pt_BR.properties

```diff
@@ -23,103 +23,103 @@
+usage.compat=Interface de Compatibilidade:\nUso: jar {ctxui}[vfmn0PMe] [jar-file] [manifest-file] [entry-point] [-C dir] arquivos] ...\nOpções:\n    -c  cria novo arquivo compactado\n    -t  lista o sumário do arquivo compactado\n    -x  extrai arquivos com o nome (ou todos) do arquivo compactado\n    -u  atualiza o arquivo compactado existente\n    -v  gera saída detalhada na saída padrão\n    -f  especifica o nome do arquivo compactado\n    -m  inclui as informações do manifesto do arquivo de manifesto especificado\n    -n   executa a normalização Pack200 após a criação de um novo arquivo compactado\n    -e  especifica o ponto de entrada da aplicativo para aplicativo stand-alone \n        empacotada em um arquivo jar executável\n    -0  armazena somente; não usa compactação ZIP\n    -P  preserva os componentes '/' inicial (caminho absoluto) e ".." (diretório pai) nos nomes dos arquivos\n    -M  não cria um arquivo de manifesto para as entradas\n    -i  gera informações de índice para os arquivos jar especificados\n    -C  passa para o diretório especificado e inclui o arquivo a seguir\nSe um arquivo também for um diretório, ele será processado repetidamente.\nO nome do arquivo de manifesto, o nome do arquivo compactado e o nome do ponto de entrada são\nespecificados na mesma ordem dos flags 'm', 'f' e 'e'.\n\nExemplo 1: para arquivar dois arquivos de classe em um arquivo compactado denominado classes.jar: \n       jar cvf classes.jar Foo.class Bar.class \nExemplo 2: use um arquivo de manifesto existente 'mymanifest' e arquive todos os\n           arquivos no diretório foo/ em 'classes.jar': \n       jar cvfm classes.jar mymanifest -C foo/ .\n
 
 main.usage.summary=Uso: jar [OPTION...] [ [--release VERSION] [-C dir] files] ...
-main.usage.summary.try=Tente `jar --ajuda' para obter mais informa\u00E7\u00F5es.
+main.usage.summary.try=Tente `jar --ajuda' para obter mais informações.
```

Comment:
I was looking for something unrelated in properties files, and found this. It is surprising to see an option name being localised; it must be a bug.

Reply:
Good catch, Pavel. It is indeed a bug. This type of overtranslation l10n bug happens all the time, and hard to catch.